u-boot-envtool build fails with gcc9

gcc9 seems to fail a build in u-boot-envtool

~~~
make[5]: Entering directory '/home/joe/local2/bench/git/gargoyle/ar71xx-src/buil
d_dir/host/u-boot-2018.03'
  HOSTCC  scripts/basic/fixdep
  CHK     include/config/uboot.release
  UPD     include/config/uboot.release
  CHK     include/generated/version_autogenerated.h
  UPD     include/generated/version_autogenerated.h
  CHK     include/generated/timestamp_autogenerated.h
  UPD     include/generated/timestamp_autogenerated.h
  HOSTCC  tools/mkenvimage.o
In file included from /usr/include/asm-generic/types.h:7,
                 from /usr/include/asm/types.h:5,
                 from /usr/include/linux/types.h:5,
                 from /usr/include/linux/stat.h:5,
                 from /usr/include/bits/statx.h:30,
                 from /usr/include/sys/stat.h:446,
                 from tools/mkenvimage.c:21:
/usr/include/asm-generic/int-ll64.h:31:42: error: conflicting types for '__u64'
   31 | __extension__ typedef unsigned long long __u64;
      |                                          ^~~~~
In file included from <command-line>:
././include/compiler.h:69:18: note: previous declaration of '__u64' was here
   69 | typedef uint64_t __u64;
      |                  ^~~~~
~~~

--- a/build.sh
+++ b/build.sh
@@ -33,7 +33,7 @@ set_version_variables()
 	# set precise commit in repo to use 
 	# you can set this to an alternate commit 
 	# or empty to checkout latest
-	openwrt_commit="f6429577c5110fe3ad5575c797e884e87027726c"
+	openwrt_commit="90f6af5108c1d81a5c7e78083d82f1035c091bd9"
 	openwrt_abbrev_commit=$( echo "$openwrt_commit" | cut -b 1-7 )
 	
 

